app/views/dvr/dvrFiles.scala.html

```diff
@@ -154,7 +155,7 @@ <h1 id="assessment-address" class="govuk-heading-l">@address</h1>
     <br/>
 
     @for(activeTabId <- model.activeTabId) {
-        <script type="text/javascript">
+        <script @{CSPNonce.attr} type="text/javascript">
         document.addEventListener("DOMContentLoaded", function() {
             location.hash = "@activeTabId";
             @if(startCheckForm.hasErrors){
```

Comment:
I've just double checked the Confluence page and I don't think we can use inline javascript anymore. I think this needs to be changed